deploy/service.yaml

```diff
@@ -4,6 +4,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: echo-service
+  namespace: keptn
```

Comment:
Actually we have agreed to not use `namespace` for manifests, such that we can easily deploy the service using `kubectl apply -f deploy/service.yaml -n my-namespace`.

This is also the way we are deploying `echo-service` here: 
https://github.com/keptn/keptn/blob/069dd0f5c7b6f37a3737f4c0c9c7cf07a801b039/test/test_linking_stages.sh#L44-L45